gen/calc.py

```diff
@@ -1319,7 +1383,20 @@ def calculate_fault_domain_detect_contents(fault_domain_detect_filename):
         'enable_mesos_ipv6_discovery': 'false',
         'log_offers': 'true',
         'mesos_cni_root_dir_persist': 'false',
-        'enable_mesos_input_plugin': 'true'
+        'enable_mesos_input_plugin': 'true',
+        'calico_enabled': 'true',
+        'calico_network_cidr': '192.168.0.0/16',
+        'calico_vxlan_enabled': 'false',
```

Comment:
I think the benefits of using ipip by default are two:
1. ipip takes less overhead than VXLAN, so it's more than efficient
2. we also use ipip as the default encapsulation method in Konvoy(, as a suggested one?) .  The benefit is that we can share some of the problems/issues in both konvoy and DC/OS.

And your concern is that we already have `vxlan` introduced in DC/OS, so using `vxlan` is much safer.  Yes, If we want to support IPIP in DC/OS, then we may face the possible problems for IPIP.

CC @GoelDeepak

Reply:
One thing to keep in mind is that DC/OS is an existing product as compare to Konvoy. With the existing product comes the existing settings. IPIP and BGP aren't allowed in some cloud environment and so an operator would have to keep in mind to set it at the time of cluster installation. `vxlan` as a default has benefits as it would not require any change in the existing DC/OS installation.

Reply:
After syncing with @urbanserj ,  IPIP in calico also suffers from scaling clusters for bird burden.  So vxlan calico + pure etcd V3 can address scaling problem, but we need to verify the cluster size, according to 
`While Typha can be used with etcd, etcd v3 is optimised to handle many clients already so we do not recommend adding Typha if you're using etcd.`
https://github.com/projectcalico/typha#when-should-i-use-typha